src/events/janus_rabbitmqevh.c

```diff
@@ -346,6 +352,26 @@ int janus_rabbitmqevh_init(const char *config_path) {
 }
 
 int janus_rabbitmqevh_connect(void) {
+	uint8_t retry_interval = 5; // Retry interval in seconds
```

Comment:
Comments should always be in `/*  */` form, not `//`. Besides, this property should probably be configurable: `5` seems very arbitrary.